### PR TITLE
Feature: Menubar

### PR DESCRIPTION
**Pull request by varffs**

### CODE REVIEW

File: app/imports/components/base/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+export class Header extends Component {
+  render() {
+    return (
+      <nav id='header' className='pt-navbar pt-dark margin-bottom-basic'>
+        <div className='pt-navbar-group pt-align-left'>
+          <div className='pt-navbar-heading'>Final Boss</div>
+        </div>
+        <div className='pt-navbar-group pt-align-right'>
+          {Roles.userIsInRole(Meteor.userId(), ['admin']) &&
```
Comment (mixcas) [inline app/imports/components/base/Header.jsx:13]: The role should be passed down from the `mainContainer` as a prop

File: app/imports/components/base/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+            <a href='/dashboard/admin' className='pt-button pt-minimal pt-icon-office' role='button'>Admin</a>
```
Comment (mixcas) [inline app/imports/components/base/Header.jsx]: We should use `FlowRouter.path()`. In the future if we need to change paths we don't need to change the markup.

File: app/imports/components/base/Header.jsx (hunk 1 above)
Comment (varffs) [reply app/imports/components/base/Header.jsx:13]: Why?

File: app/imports/components/base/Header.jsx (hunk 2 above)
Comment (varffs) [reply app/imports/components/base/Header.jsx]: yh totally forgot this practice

**Pull request merged**